libcloud/compute/drivers/ec2.py

```diff
@@ -850,6 +850,26 @@ def GiB(value):
             'cpu': 8
         }
     },
+    'x1.16xlarge': {
+        'id': 'x1.16xlarge',
+        'name': 'Memory Optimized Sixteen Extra Large instance',
+        'ram': GiB(976),
+        'disk': 1920,  # GB
+        'bandwidth': None,
+        'extra': {
+            'cpu': 64
+        }
+    },
+    'x1e.32xlarge': {
+        'id': 'x1e.32xlarge',
+        'name': 'Memory Optimized ThirtyTwo E Extra Large instance',
```

Comment:
Small typo, but I will fix it when merging.

Reply:
Nvm, it's not actually a typo - that's a correct name.

Please ignore my comment.